.github/workflows/ci_cd.yml

```diff
@@ -175,3 +183,23 @@ jobs:
           docker tag ${{ matrix.image }} \
             ghcr.io/wordpress/openverse-${{ matrix.image }}:${{ github.ref_name }}
           docker push --all-tags ghcr.io/wordpress/openverse-${{ matrix.image }}
+
+  docs:
+    name: Publish developer docs
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && github.repository == 'WordPress/openverse-api'
```

Comment:
In what cases would the `github.repository` be something different? Forks, possibly?

Reply:
Yeah, I suppose the same. Tbh, I didn't think too much about it and just copied @sarayourfriend's work on deploying the Storybook in `openverse-frontend`.

Reply:
Got it! That makes sense :) It just looked, at first glance, very redundant haha